src/mainMenu/mainMenu.js

```diff
@@ -0,0 +1,94 @@
+class MainMenu {
+
+    constructor() {
+        this.createMainMenu();
+        this.addListeners();
+    }
+
+    creatingButtonContainer (name) {
+        const mainMenuContainer = document.createElement('div');
+        mainMenuContainer.id = `${name}`;
+        mainMenuContainer.style=
+            `
+            background-color: #fff; 
+            margin = 1rem; 
+            border-radius: 2rem; 
+            box-shadow: 0 0 25px 1px red;
+            display: flex;
+            justify-content: space-around;
+            padding: 4rem;
+            margin-bottom: 10rem;
+            `
+        document.querySelector('.gameArea').appendChild(mainMenuContainer);
+    };
+    
+    creatingButton(name) {
+        let button = document.createElement("button");
+        let buttonH2 = document.createElement('h2');
+        let buttonDes = document.createTextNode(`${name}`);
+        button.id = `button${name}`;
+        buttonH2.appendChild(buttonDes);
+        button.appendChild(buttonH2);
+        button.classList.add("button", "gameModeButton", "hoverCursor");
+        button.setAttribute("isactive", "false");
+    
+        button.style=
+        `
+        background-color: #fff; 
+        border: none; 
+        color: #e6e6e6; 
+        `;
+    
+        document.querySelector('#mainMenu').appendChild(button);
+    };
+    
+    puttingCorrectColor() {
+        const isActivePeople = buttonPeople.getAttribute("isActive");
+        const isActiveVehicle = buttonVehicles.getAttribute("isActive");
+        const isActiveStarships = buttonStarships.getAttribute("isActive");
+        
+        isActivePeople === "true" ? buttonPeople.style.color = "#000" : buttonPeople.style.color = "#e6e6e6";
```

Comment:
you don't have to compare with === "true", instead you can use:
Boolean(isActivePeople) ? .... oraz !!isActivePeople ? ...